src/.eslintrc.js

```diff
@@ -27,13 +27,19 @@ module.exports = {
             // Disallow importing AMP dependencies into preact/Bento
             'target': 'src/preact',
             'from': 'src',
-            'except': ['./core', './preact'],
+            'except': [
+              './core',
+              './preact',
+              './shadow-embed',
+              './3p-frame',
+              './url',
+            ],
           },
           {
             // Disallow importing non-core dependencies into polyfills
             'target': 'src/polyfills',
             'from': 'src',
-            'except': ['./core', './polyfills'],
+            'except': ['./core', './polyfills', './utils/log'],
```

Comment:
This file prevents us from accidentally pulling in large subsystems, ex. importing the entire logging system by mistake in certain areas of code that shouldn't, such as code that may be bundled with Bento components where we don't want all the bytes of a logging system we shouldn't use. Individual files with "illegal" imports, like `#polyfills/fetch` which depends on `#utils/log`, are excluded in the `overrides` section below. They don't do anything in CI, but they give a warning locally. This is WAI, and a fresh `main` checkout and `amp lint` run seems to indicate that all is as it should be and there are no new errors.